cmd/make-virt-host/main.go

```diff
@@ -156,7 +156,7 @@ func main() {
 	}
 
 	domainResult := Domain{}
-	xml.Unmarshal([]byte(virshOut), &domainResult)
+	xml.Unmarshal(virshOut, &domainResult)
```

Comment:
What was the reasoning here?

Reply:
Unnecessary type casting, virshOut is already a `byte` array.